docs/utils/nft-borsh.ts

```diff
@@ -137,3 +137,19 @@ export const NFTOKEN_COLLECTION_ACCOUNT = new FixableGlowBorsh<{
     ["metadata_url", FixableGlowBorsh.utf8String],
   ],
 });
+
+export const SYSTEM_CREATE_ACCOUNT_IX = new FixableGlowBorsh<{
```

Comment:
Let's copy over what we already have in Luma rather than re-invent this

Reply:
Ah cool, I just didn't know we had it

Reply:
I see, this one right? https://github.com/luma-team/luma